Tweak.xm

```diff
@@ -591,6 +657,28 @@ void initFlipSwitch()
             %init(ControlCenter, BUTTONCLASS=(objc_getClass("SBControlCenterButton") ?: objc_getClass("CCUIControlCenterPushButton")));
         }
         
+        BOOL shouldInit = NO;
+        if (!NSClassFromString(@"APTableCell")) {
```

Comment:
Can you explain the logic here? This is not obvious to me. Why do you load the AirPort Settings bundle, if it succeeds, wait for the network bundle to be loaded?

Reply:
As with the notification method above, you need to prepare another method.